plugins/backstage-plugin-coder/src/components/CoderAuthWrapper/CoderAuthInputForm.tsx

```diff
@@ -95,12 +95,21 @@ export const CoderAuthInputForm = () => {
     registerNewToken(newToken);
   };
 
+  const formHeaderId = `${hookId}-form-header`;
   const legendId = `${hookId}-legend`;
   const authTokenInputId = `${hookId}-auth-token`;
   const warningBannerId = `${hookId}-warning-banner`;
 
   return (
-    <form className={styles.formContainer} onSubmit={onSubmit}>
+    <form
+      aria-labelledby={formHeaderId}
+      className={styles.formContainer}
+      onSubmit={onSubmit}
+    >
+      <h3 hidden id={formHeaderId}>
+        Authenticate with Coder
+      </h3>
```

Comment:
This was missing from the component before – has two purposes:
- Increasing accessibility for the overall form
- Providing another way to hook into the component during testing